### ISSUE TITLE
Unable to update macOS Monterey 12.0.1 to 12.2

### ISSUE THREAD

**Issue by maksumon** [chris1111/HP-Probook-EliteBook-Package-Creator-OC#121]
@chris1111 Thanks for this excellent package. I have successfully installed macOS Monterey 12.0.1 on my HP ProBook 4540s. But unable to update to the latest version which is 12.2. Do you have any suggestions on this, please?
![Screenshot 2022-02-03 at 2 59 17 AM](https://user-images.githubusercontent.com/1089462/152220251-6cb4a2d5-c930-45ad-96d1-0b3b0569aed3.png)

**Comment by chris1111** [comment]
Click advance thème sait and see

**Comment by maksumon** [comment]
Tried more than 15 times but after checking for updates it is still the same.

![Screenshot 2022-02-03 at 7 07 52 PM](https://user-images.githubusercontent.com/1089462/152331458-7f599e32-8132-43bb-8c6c-30be6ff39e43.png)

**Comment by chris1111** [comment]
it is strange
try to reset nvram OpenCore boot menue

**Comment by maksumon** [comment]
I tried that too before... Let me try again in few minutes

**Comment by chris1111** [comment]
You can try disable Auto Update ?

**Comment by maksumon** [comment]
Tried both resetting NVRam and disabling auto-update but no luck...

**Comment by chris1111** [comment]
Try to delete /Library/ Updates/index.plist /ProductMetadata.plist
Take backup of the file before
delete them then reboot / retry

**Comment by maksumon** [comment]
Unfortunately, still the same. Should I try updating the Open Core Legacy Patcher?

**Comment by maksumon** [comment]
I tried to get update from the AppStore and getting this error:
![Screenshot 2022-02-03 at 8 25 59 PM](https://user-images.githubusercontent.com/1089462/152342598-7d3b78ca-8ec3-4f18-b814-fe224740fc5d.png)

**Comment by maksumon** [comment]
@chris1111 Oops, sorry my bad. I had changed the SMBIOS info thus no update was available... Thanks for supporting me in diagnosing this issue

**Comment by chris1111** [comment]
OCLP as nothing to do here 
Probably why thaïs issue

**Comment by maksumon** [comment]
![Screenshot 2022-02-03 at 10 39 16 PM](https://user-images.githubusercontent.com/1089462/152364429-e711e1f4-cc53-4b94-8c0b-c524f3c8e725.png)

**Comment by chris1111** [comment]
[Solved ☞  (Never use OpenCore Legacy Patcher with this Program)](https://github.com/chris1111/HP-Probook-EliteBook-Package-Creator-OC/issues/121#issuecomment-1029016942)
